CHANGELOG.md

```diff
@@ -2,7 +2,54 @@
 
 All notable changes to this project will be documented in this file.
 
-## Unreleased [(diff)][unreleased-diff]
+## [0.3.0] UNRELEASED [(diff with 0.2.1)][unreleased-diff]
```

Comment:
Somewhere we should document how were going to change the pre-release version number for breaking and non-breaking changes.

Reply:
we're going to treat all prereleases of breaking changes, otherwise we might as well just make normal releases.